keyboards/planck/keymaps/motform/keymap.c

```diff
@@ -0,0 +1,239 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/* ABOUT THIS LAYOUT
+ * 
+ * Implements colemak-se: github.com/motform/colemak-se
+ * Optimized for use with Emacs and Evil-mode
+ * Only vetted for the MacOS implementation of Swedish QWERTY
+ * Don’t forget to turn of dead keys!
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+#include "keymap_swedish.h"
+
+extern keymap_config_t keymap_config;
+
+enum planck_layers {
+  _COLEMAK,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum planck_keycodes {
```

Comment:
If you're only using the one default layer, and you're not using the backlight, then you can remove this enum, completely, then.